[Fix] Ignore gamescope if enabled and not found instead of failing

arielj opened the pull request:
This PR fixes https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/3341

Instead of failing with an error, ignore the `gamescope` setup if enabled but not found.

This can happen if gamescope was configured for a game and then uninstalled.

With this change it will just show a warning in the logs and still run the game as if the setting was disabled.

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

Etaash-mathamsetty reviewed (approved):
this diff is hard to read but if I read it right it looks fine

arielj commented:
> this diff is hard to read but if I read it right it looks fine

yeah, github is trying to be too smart, I just changed the return with a `logWarning` and moved the rest into an `else` (just added indentation to most of the lines there), github is going crazy but there's no other change on that part of the code apart of the extra indentation
